### DISCUSSION TITLE
FR : Automatically Login

### DISCUSSION THREAD

**Discussion by kuhnino** [mealie-recipes/mealie#2310] [Feature Request]
I believe the webinterface may need an automatic login function for my users.

Currently I always have to press "login" rather then the UI keeps me logged in

Seen on Firefox

**Comment by michael-genson** [comment]
Can you post your deployment details, and on what device/devices you're seeing this issue?

**Comment by kuhnino** [comment]
Sure

Details about my installation :
- Mealie as Docker Image
- Running on UnRaid
- Client Windows 11
- Browser Firefox 111.0.1 should be latest

Let me add some more details about the FR description
- Whenever I open Mealie WebUI, I´m not logged in
- I always have to click on LOGIN in top right corner
- then my login credentials stored in Browser are automaticaly entered in upcoming popup
- only have to press then LOGIN and I´m in

As I believe it would be nice to keep me logged in, no?

**Comment by JvdMaat** [comment]
Do you have the TOKEN_TIME set in docker?
```
    environment:
      - PUID=1000
      - GUID=1000
      - TZ=${TZ}
      - TOKEN_TIME=40000 #hours?
```
I have mine set to 40000 hours (4.5 years) to just take that out of the picture. I'm not sure what the default is.

**Comment by michael-genson** [comment]
What version of Mealie are you running?

**Comment by kuhnino** [comment]
Good question 
0.5.6 from the UNRAID CONTAINER APP STORE

**Comment by kuhnino** [comment]
Token Time? No not set
I assume it's
TOKEN_TIME=40000
without the #hours?

**Comment by JvdMaat** [comment]
> Token Time? No not set I assume it's TOKEN_TIME=40000 without the #hours?

Yes, sorry. There was no documentation and that was my comment on what I assumed the unit was.  (I use docker compose, so I leave notes in there for reference on certain items)

**Comment by michael-genson** [comment]
> Good question 0.5.6 from the UNRAID CONTAINER APP STORE

The `TOKEN_TIME` env var works in 0.5.6, but you may want to consider migrating to 1.0.0b; the version you're on is very outdated: https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/

> Yes, sorry. There was no documentation and that was my comment on what I assumed the unit was

FYI it's in the backend configuration documentation: https://nightly.mealie.io/documentation/getting-started/installation/backend-config/

And for the old 0.5.6 version: https://hay-kot.github.io/mealie/documentation/getting-started/install/

**Comment by kuhnino** [comment]
Hm,
I was looking here:
![grafik](https://user-images.githubusercontent.com/77860530/228854223-190e9c1a-e8d4-4be0-a0f6-399c0eafcb06.png)
https://hay-kot.github.io/mealie/changelog/v0.5.6/

so thought this version is new.

Anyhow, will try to upgrade and see if that is getting fixed.

BTW TX for your help

**Comment by michael-genson** [comment]
No problem! The `TOKEN_TIME` setting should help either way (0.5.6 or 1.0.0b). There were some issues with the PWA on the beta version, but if you upgrade to nightly they're fixed now

**Comment by kuhnino** [comment]
Great, i will use the TOKEN TIME and wait for release. I do have some concersn using nightly builds.
Don´t know why, mabe related to my" vampire allergy" ;-))))))
